rr_common/src/image_transformation/pointcloud_projector.cpp

```diff
@@ -68,10 +78,34 @@ class PointCloudProjector : public nodelet::Nodelet {
         pointcloud_pub_.publish(out);
     }
 
+    void axes_callback(const rr_msgs::axes& ax) {
+        if (!axes_init) {
+            // Set the initial axes
+            tf2::convert(cam_geom_.GetCameraPose().orientation, initial_axes);
+        }
+
+        // Convert to correct Quaternion type
+        tf2::Quaternion quat_tf;
+        tf2::convert(cam_geom_.GetCameraPose().orientation, quat_tf);
+
+        // Set roll, pitch, and yaw
+        quat_tf.setRPY(ax.roll, ax.pitch, ax.yaw);
+        // Calculate the DELTA of the pitch and roll
+        quat_tf -= initial_axes;
+
+        // Convert back
+        cam_geom_.GetCameraPose().orientation = tf2::toMsg(quat_tf);
```

Comment:
GetCameraPose returns a Pose by value, so changing this orientation does not change the internal state of cam_geom_